internal/lookout/ui/src/services/JobService.ts

```diff
@@ -225,6 +243,82 @@ export default class JobService {
     return result
   }
 
+  async reprioritizeJobs(jobs: Job[], newPriority: number): Promise<ReprioritizeJobsResult> {
+    const result: ReprioritizeJobsResult = { reprioritizedJobs: [], failedJobReprioritizations: [] }
+    const jobIds: string[] = []
+    for (const job of jobs) {
+      jobIds.push(job.jobId)
+    }
+    try {
+      const apiResult = await this.submitApi.reprioritizeJobs({
+        body: {
+          jobIds: jobIds,
+          newPriority: newPriority,
+        },
+      })
+
+      for (const job of jobs) {
+        if (apiResult.reprioritizationResults?.hasOwnProperty(job.jobId)) {
+          const error = apiResult.reprioritizationResults[job.jobId]
+          if (error === "") {
+            result.reprioritizedJobs.push(job)
+          } else {
+            result.failedJobReprioritizations.push({ job: job, error: error })
+          }
+        } else {
+          result.reprioritizedJobs.push(job)
+        }
+      }
+    } catch (e) {
+      console.error(e)
+
+      const errorMessage = await getErrorMessage(e)
+      for (const job of jobs) {
+        result.failedJobReprioritizations.push({ job: job, error: errorMessage })
+      }
+    }
+    return result
+  }
+
+  async reprioritizeJobSets(queue: string, jobSets: JobSet[], newPriority: number): Promise<ReprioritizeJobSetResult> {
+    const result: ReprioritizeJobSetResult = { reprioritizedJobSets: [], failedJobSetReprioritizations: [] }
+
+    for (const jobSet of jobSets) {
+      try {
+        const apiResult = await this.submitApi.reprioritizeJobs({
+          body: {
+            queue: queue,
+            jobSetId: jobSet.jobSetId,
+            newPriority: newPriority,
+          },
+        })
+        let errorCount = 0
+        let successCount = 0
+        let error = ""
+        for (const key in apiResult.reprioritizationResults) {
```

Comment:
Worth considering replacing with: 
`for (const [key, error] of Object.entries(apiResult.reprioritizationResults))`